frontend/packages/db-structure/src/parser/drizzle/mysql/__tests__/index.test.ts

```diff
@@ -0,0 +1,87 @@
+import { describe, expect, it } from 'vitest'
+import { processor as _processor } from '../index.js'
```

Comment:
I'm checking now. `as _processor` needed?

Reply:
Thank you for pointing this out. This was also an unnecessary definition.
https://github.com/liam-hq/liam/pull/2642/commits/16c3d112e44a2dc34e97c32d5e845923d68238d1